bump.sh

```diff
@@ -55,8 +55,17 @@ if [[ $version == "patch" ]]; then
   sed -i "s|$version_header|\1|" ./CHANGELOG.md
 fi
 
-# Add a section to the changelog that state the version of CDK being used
 version_header="# \[$new_version](.*) (.*)"
+
+# Add a section to the changelog that states the supported Deadline versions
+DEADLINE_RELEASE_NOTE_URL="https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/release-notes.html"
+DEADLINE_SUPPORTED_VERSIONS=$(node ./scripts/getSupportedDeadlineVersions.ts)
```

Comment:
I'm surprised this works without using `ts-node` since it's pointing to a `.ts` file.